### PR TITLE
Travis integration

### PR DESCRIPTION
**Pull request by Chris-Schnaufer**
Adding TravisCI integration for extractors.

I am keeping this request open until August 20, 2019 - 5:00 PM Pacific

### CODE REVIEW

**Comment by dlebauer** [comment]
do we want to have 50MB files in the repo? are there other options?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# Drone Pipeline
```
Comment (dlebauer) [inline README.md:1]: link to osf

Comment (Chris-Schnaufer) [reply README.md:1]: Done

**Comment by Chris-Schnaufer** [comment]
> do we want to have 50MB files in the repo? are there other options?

Access to Google Drive has been available: updated to only use that

**Pull request merged**